packages/uplosi.conf.template

```diff
@@ -0,0 +1,11 @@
+[base]
+imageVersion = "0.0.1"
+name = "contrast"
+provider = "azure"
+
+[base.azure]
+subscriptionID = "@@SUBSCRIPTION_ID@@"
+location = "@@LOCATION@@"
+resourceGroup = "@@RESOURCE_GROUP@@"
+sharedImageGallery = "contrast"
+sharingProfile = "private"
```

Comment:
Did you check this works with private images? Otherwise public is acceptable, too.

Reply:
Yes, I did, I created a cluster with this, and worked. I ran into some error when I tried to create a public gallery, but didn't look much into it.